packages/core/src/network/heartbeat.spec.ts

```diff
@@ -16,9 +16,9 @@ describe('unit test heartbeat', async () => {
     interact: sinon.fake.resolves(true)
   } as any
 
-  beforeEach(() => {
+  beforeEach(async () => {
     clock = sinon.useFakeTimers(Date.now())
-    peers = new NetworkPeerStore([])
+    peers = new NetworkPeerStore([], await PeerId.create({ keyType: 'secp256k1' }))
```

Comment:
Why does the keyType matter in a test?

Reply:
it shouldn't - if there is no keytype given, It uses RSA and runs into the whole prime selection logic which burns a lot of resources.

Reply:
ok